### DISCUSSION TITLE
Run "Add all missing import" on format

### DISCUSSION THREAD

**Discussion by giacomoguidotto** [zed-industries/zed#21296] [Feature Requests]
I have the following setting:

```
"code_actions_on_format": {
    "source.organizeImports": true,
    "source.addAllMissingImports": true,
    "source.removeUnusedImports": true
  },
```
but addAllMissingImports doesn't work. Is it the right code actions?
Also, is there a way to retrieve the list of all the code actions codenames?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
